README.md

```diff
@@ -12,11 +12,44 @@ cd $GOPATH/src/sdx-encrypted-submitter
 go get -u github.com/streadway/amqp
 go get -u gopkg.in/yaml.v2
 go build
+go install
```

Comment:
The go get command is incorrect in the README, it has an _ instead of -

It should be: 

go get github.com/ONSdigital/sdx-encrypted-submitter

Reply:
Thanks , done